zipper.go

```diff
@@ -73,7 +73,11 @@ func NewZipper(name string, meshConfig map[string]config.Downstream, options ...
 		)
 		downstream := core.NewClient(name, core.ClientTypeUpstreamZipper, clientOptions...)
 
-		server.Logger().Info("add downstream", "name", downstreamName, "addr", addr, "downstream_id", downstream.ClientID())
+		server.Logger().Info(
+			"add downstream",
+			"name", downstreamName, "addr", addr,
+			"downstream_id", downstream.ClientID(), "downstream_name", downstream.Name(),
```

Comment:
I've take a try in `example-4` and got this from `zipper-1`:

<img width="600" alt="image" src="https://github.com/yomorun/yomo/assets/65603/dee30cc3-16a9-4a9a-99aa-343d99fee743">

`downstram_name` and `downstream_id` seems wrong.

please re-check.